kubernetes-deploy.gemspec

```diff
@@ -24,7 +24,7 @@ Gem::Specification.new do |spec|
 
   spec.required_ruby_version = '>= 2.3.0'
   spec.add_dependency "activesupport", ">= 5.0"
-  spec.add_dependency "kubeclient", "~> 2.4"
+  spec.add_dependency "kubeclient", "~> 3.0"
   spec.add_dependency "rest-client", ">= 1.7" # Minimum required by kubeclient. Remove when kubeclient releases v3.0.
```

Comment:
Per the comment, is this needed anymore?

Reply:
Nope fixed upstream https://github.com/abonas/kubeclient/blob/v3.0.0/kubeclient.gemspec#L30